### DISCUSSION TITLE
Img2img, TypeError: string indices must be integers.

### DISCUSSION THREAD

**Discussion by ZeroCool22** [hlky/stable-diffusion-webui#660] [Q&A] [answered]
![Screenshot_3](https://user-images.githubusercontent.com/13344308/188364681-b4c57ccc-1dec-4ccb-b1d5-046e820d8bd9.jpg)

What it means?

**Comment by mchaker** [comment, marked as answer]
This is being fixed in an existing PR

**Reply by mchaker** [reply]
There is a fix in the `img2img-fix` branch. See comment: https://github.com/hlky/stable-diffusion-webui/issues/664#issuecomment-1237025304
